src/BaGet.Core/Search/DependentsResponse.cs

```diff
@@ -19,6 +19,36 @@ public class DependentsResponse
         /// The package IDs matched by the dependent query.
         /// </summary>
         [JsonPropertyName("data")]
-        public IReadOnlyList<string> Data { get; set; }
+        public IReadOnlyList<DependentResponse> Data { get; set; }
+    }
+
+    /// <summary>
+    /// A package that depends on the queried package.
+    /// </summary>
+    public class DependentResponse
+    {
+        /// <summary>
+        /// The dependent package key.
+        /// </summary>
+        [JsonPropertyName("key")]
+        public int Key { get; set; }
```

Comment:
What's the key for? Can we remove it?

Reply:
I guess we don't technically need it, I was going to use it as the collection key in the UI?